crates/chia-datalayer/src/merkle.rs

```diff
@@ -0,0 +1,1997 @@
+#[cfg(feature = "py-bindings")]
+use pyo3::{
+    buffer::PyBuffer,
+    exceptions::{PyAttributeError, PyValueError},
+    pyclass, pymethods, PyResult, Python,
+};
+
+use clvmr::sha2::Sha256;
+use num_traits::ToBytes;
+use std::cmp::Ordering;
+use std::collections::{HashMap, HashSet, VecDeque};
+use std::iter::zip;
+use std::mem::size_of;
+use std::ops::Range;
+
+type TreeIndex = u32;
+type Parent = Option<TreeIndex>;
+type Hash = [u8; 32];
+type KvId = i64;
```

Comment:
I'm not sure what this is for, maybe a comment explaining it and/or a newtype with a doc comment would be appropriate for this as well.

Reply:
newtype at [`58c79a2` (#673)](https://github.com/Chia-Network/chia_rs/pull/673/commits/58c79a290017a08030a60c427a1f7bd0ce891be5)